src/com/lftechnology/java/training/krishna/basics/smallest_multiple/SmallestMultipleDemo.java

```diff
@@ -0,0 +1,16 @@
+package com.lftechnology.java.training.krishna.basics.smallest_multiple;
+
+import java.util.logging.Logger;
+
+public class SmallestMultipleDemo {
```

Comment:
class level documentation missing here too.. please avoid repeating such scenario time and again